TravelPro.css

```diff
@@ -0,0 +1,490 @@
+body{
```

Comment:
/* Instead of this: */
margin-top: 20px;
margin-right: 20px;
margin-bottom: 20px;
margin-left: 20px;

/* Use this: */
margin: 20px;